worlds/archipidle/Rules.py

```diff
@@ -14,26 +14,23 @@ def _archipidle_location_is_accessible(self, player_id, items_required):
 
 
 def set_rules(world: MultiWorld, player: int):
-    for i in range(1, 16):
-        set_rule(
-            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) > 0 else 0} seconds", player),
-            lambda state: state._archipidle_location_is_accessible(player, 0)
-        )
-
     for i in range(16, 31):
         set_rule(
-            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) > 0 else 0} seconds", player),
+            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) else 0} seconds", player),
             lambda state: state._archipidle_location_is_accessible(player, 4)
         )
 
     for i in range(31, 51):
         set_rule(
-            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) > 0 else 0} seconds", player),
+            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) else 0} seconds", player),
             lambda state: state._archipidle_location_is_accessible(player, 10)
         )
 
     for i in range(51, 101):
         set_rule(
-            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) > 0 else 0} seconds", player),
+            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) else 0} seconds", player),
             lambda state: state._archipidle_location_is_accessible(player, 20)
         )
+
+    world.completion_condition[player] = lambda state: \
+        state.can_reach(world.get_location(f"IDLE for at least 50 minutes 0 seconds", player), "Location", player)
```

Comment:
is this really necessary? i mean archipidle is always beatable from the start, right?

Reply:
yeah that's fair will revert this.